bubblegum/program/src/tests/mod.rs

```diff
@@ -0,0 +1,712 @@
+mod simple;
+
+use std::cell::{RefCell, RefMut};
+use std::convert::TryFrom;
+use std::mem::size_of;
+use std::result;
+
+use anchor_lang::{self, AccountDeserialize, InstructionData, ToAccountMetas};
+use bytemuck::{try_from_bytes, PodCastError};
+use solana_program::instruction::Instruction;
+use solana_program::pubkey::Pubkey;
+use solana_program::rent::Rent;
+use solana_program::{keccak, system_instruction, system_program};
+use solana_program_test::{BanksClient, BanksClientError, ProgramTest};
+use solana_sdk::account::Account;
+use solana_sdk::signature::{Keypair, Signer, SignerError};
+use solana_sdk::signer::signers::Signers;
+use solana_sdk::transaction::Transaction;
+use spl_compression::state::ConcurrentMerkleTreeHeader;
+use spl_concurrent_merkle_tree::concurrent_merkle_tree::ConcurrentMerkleTree;
+
+use crate::state::metaplex_adapter::MetadataArgs;
+use crate::state::request::MintRequest;
+use crate::state::TreeConfig;
```

Comment:
Why is it jarring?  I actually find the combined ones easier to read because the whitespace and nesting makes the relationships clearer.  But this is probably an unsolvable debate where half of people like one way and half of people like the other way.

Reply:
Sometimes, when multiple levels get aggregated on single line, it becomes much more difficult to scan for a particular item when you want to leverage stuff like the `use`s being sorted; however I don't feel super strongly either way and I'll gladly conform to the convention currently in use. Also, I think I missed unifying one or two paths, will fix.

Reply:
ok I see, sounds good, thanks!